src/app/cli.h

```diff
@@ -152,6 +152,7 @@ struct CliParameters {
             }
         }
 
+        filenames.reserve( static_cast<size_t>( parser.positionalArguments().count() ) );
```

Comment:
Usually there are not that many files on command line. Reserve here not needed.

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3